### PR TITLE
Random graph

### PR DESCRIPTION
**Pull request by pstjohn**
Had some issues in `untrained_model_sampling.ipynb` that I haven't figured out; but the tests seem to pass and that issue was with the GNN-state

### CODE REVIEW

File: experiments/tsp/run_tsp.py [hunk 1]
```diff
@@ -123,17 +125,20 @@
     # Define custom_model, config, and state based on GNN yes/no
     if args.use_gnn:
         custom_model = "TSPGNNModel"
-        custom_model_config = {"num_messages": 1, "embed_dim": 32}
+        custom_model_config = {"num_messages": 3, "embed_dim": 32}
```
Comment (pstjohn) [inline experiments/tsp/run_tsp.py:128]: @davebiagioni, I think this was the most consequential change. we could even experiment with larger models if we scaled up the rollouts

File: experiments/tsp/run-tsp.sh [hunk 2]
```diff
@@ -12,14 +11,14 @@ clear
 source env.sh
 python run_tsp.py \
     --N=40 \
-    --use-gnn \
     --run=PPO \
+    --use-gnn \
     --lr=0.001 \
     --entropy-coeff=0.01 \
     --seed=0 \
+    --rollouts-per-worker=10 \
```
Comment (pstjohn) [inline experiments/tsp/run-tsp.sh:19]: not sure this changed anything

Comment (davebiagioni) [reply experiments/tsp/run-tsp.sh:19]: so this should increase your sampling time by 10x from the default value of 1 -- but could lead to faster convergence if gradients are better estimated.  always a tricky one to tune, and of course for constant train_batch_size you'd have to scale it with number of workers

**Comment by pstjohn** [comment]
So just some thoughts on how we could wrap this up; I'm happy to implement the cost rescaling to make convergence more obvious. In terms of other comparisons, Concorde gives the exact solution; so i suppose could compute an optimality gap there if we wanted.

Do we want to increase N to some nontrivial size, or is 40 large enough to be demonstrative? Do we want to run a comparison of different RL algorithms from rllib?

I think it would be great to quantitatively show our results are "competitive" with the current literature, however we want to define that

**Pull request merged**